index.html

```diff
@@ -3,119 +3,124 @@
+    <div class="hero__container">
+      <div class="nav-flex">
+        <h1 class="nav-group"><a href="#">Dustin Walker</a></h1>
+        <ul class="nav-group">
+          <li class="nav-group__item"><a href="#">Home</a></li>
+          <li class="nav-group__item"><a href="#">About</a></li>
+          <li class="nav-group__item"><a href="#">Contact</a></li>
         </ul>
       </div>
     </div>
   </nav>
 
-  <div class="z__fill flex-v-center">
-    <div class="z__container">
+  <div class="hero__fill">
+    <div class="hero__container">
       <h2>
         This is my portfolio!
-      h2>
+      </h2>
       <h2>
-        There may be many like it, so I should replace this text to show that I am reading.
+        I am in fact reading.
       </h2>
       <h2>
-        <a href="mailto:my-email@goes-here.com" class="is-this-a-link">Contact Me?
+        <a href="mailto:dwtimp@gmail.com" class="btn ">Contact Me?</a>
       </h2>
     </div>
   </div>
 </header>
 
-<section class="y y--a">
-  <div class="y__container">
-    <h2 class="title title--jumbo">I'm a student at the Iron Yard</h2>
+<section class="section section--info">
+  <div class="section__header">
+    <h2 class="fancy-text--big">I'm a student at the Iron Yard</h2>
 
-    <p class="title title--normal">I should write something about myself here and make it crafty.</p>
+    <p class="fancy-text">I go to a lot of concerts.</p>
   </div>
 </section>
 
-<section class="y>
-  <div class="y__container">
-    <h3 class="y__header">My First Day</h3>
+<section class="section">
+  <div class="section__container">
+    <h3 class="section__header">My First Day</h3>
 
-    <hr class="line">
+    <hr class="divider">
 
-    <p>Day one is hard. It should be tough and challenging. I'll write something about it here. Two paragraphs would be nice.</p>
+    <p class="fancy-text">Day one is hard. It should be tough and challenging. Getting used to the new surroundings, new people, new material
+      is a little overwhelming. Started to feel more comfortable with things very quickly though.</p>
   </div>
 </section>
 
-<section class="y y--b">
-  <div class="y__container">
-    <h2 class="title title--jumbo">This is my tagline</h2>
+<section class="section section--photo">
+  <div class="section__header">
+    <h2 class="fancy-text--big">Tired, but wiser for the time</h2>
   </div>
 </section>
 
-<section classy y--c">
-  <div class="y__container">
-    <div class="notsure">
-      <div class="notsure__yeah notsure__yeah-?-?">
-        <div class="c">
-          <figure class="instagram">
-            <img src="./dj.jpeg" alt="Replace this with another image please">
+<section class= "section section--more">
+  <div class="section__container">
+    <div class="row">
+      <div class="row__column row__column-1-3">
+        <div class="card">
+          <figure class="polaroid">
+            <img src="./image/dj.jpeg" alt="DeeJay">
           </figure>
+            </div>
+          </div>
```

Comment:
This div closes the `row` too early so the picture isn't next to the icon cards.  Move this closing tag to the line 108ish (near the end of this current `section`).